resources/test/fixtures/flake8_commas/COM81.py

```diff
@@ -383,7 +383,7 @@ def func(
 
 s2 = (
     'this'
-    'is a also a string'
+    'is also a string'
```

Comment:
Since this is a change to the test fixture, it would probably also need the equivalent snapshot change.

Reply:
Sometimes :) It depends on whether the changed code is part of a violation, and whether it impacts the line and column numbers of the violation. In this case, this code is _correct_ to omit a trailing comma, I think, so there's no impact.